src/features/ITGovernance/Admin/GRBReview/BusinessCaseCard/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  Icon
+} from '@trussworks/react-uswds';
+import classNames from 'classnames';
+
+import Alert from 'components/Alert';
+import {
+  DescriptionDefinition,
+  DescriptionList,
+  DescriptionTerm
+} from 'components/DescriptionGroup';
+import UswdsReactLink from 'components/LinkWrapper';
+import { BusinessCaseModel } from 'types/businessCase';
+import { formatDateLocal } from 'utils/date';
+
+export type IntakeRequestCardProps = {
+  businessCase: BusinessCaseModel;
+  className?: string;
+};
+
+const BusinessCaseCard = ({
+  businessCase,
+  className
+}: IntakeRequestCardProps) => {
+  const { t } = useTranslation('intake');
```

Comment:
Translations not coming through:

![23947](https://github.com/user-attachments/assets/6d2e4105-33f4-4272-8b4e-6ced7322694a)

Reply:
Oh snapppp!